ras/supportconfig.py

```diff
@@ -238,5 +230,5 @@ def run_supportconfig(self):
         self.is_fail = 0
         process.run("supportconfig", sudo=True, ignore_status=True)
         if self.is_fail >= 1:
-            self.fail(
-                "%s command(s) failed in sosreport tool verification" % self.is_fail)
+            self.fail("%s command(s) failed in supportconfig tool verification"
+                      % self.is_fail)
```

Comment:
i see some of style changes is it possible to that as part of other commit please